### ISSUE TITLE
error installing wooey via pip

### ISSUE THREAD

**Issue by andrewcstewart** [wooey/Wooey#59]
```
$ pip install wooey
Collecting wooey
  Using cached Wooey-0.0.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-fQuuVd/wooey/setup.py", line 3, in <module>
        with open('README.md') as readme:
    IOError: [Errno 2] No such file or directory: 'README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-fQuuVd/wooey
```

**Comment by mfitzp** [comment]
@andrewcstewart currently the PyPi repo is broken. Can you try installing from this repo instead?

    pip install git+https://github.com/wooey/Wooey.git

...will work (if you have git installed).

We'll have a release up on PyPi in the next few days.

**Comment by andrewcstewart** [comment]
Of course.  I just didn’t see an issue open for the PyPi error so thought I’d report it.

**Comment by Chris7** [comment]
Fixed with 0.9.0 update.
